ibis/tests/sql/snapshots/test_sqlalchemy/test_no_cross_join/out.sql

```diff
@@ -1,15 +1,15 @@
 SELECT
-  t0.id AS id_x,
-  t0.personal,
-  t0.family,
-  t1.taken,
-  t1.person,
-  t1.quant,
-  t1.reading,
+  t3.id AS id_x,
+  t3.personal,
+  t3.family,
+  t4.taken,
+  t4.person,
+  t4.quant,
+  t4.reading,
   t2.id AS id_y,
   t2.site,
   t2.dated
-FROM person AS t0
```

Comment:
Ok, this should be fixed now. The issue was that we still need to look up references to tables, using the parent context in case we're referring to them in a child context.

Reply:
Down to 11 files changed!